protocol/src/main/java/org/apache/drill/exec/proto/UserProtos.java

```diff
@@ -5798,6 +5798,34 @@ public static UserToBitHandshake getDefaultInstance() {
     public UserToBitHandshake getDefaultInstanceForType() {
       return defaultInstance;
     }
+    public String safeLogString() {
```

Comment:
You cannot add custom methods to proto buffers. Also consider using tabs instead of multiple spaces.
Please add to Jira example how log files looked before your changes and after.